ocean_lib/data_provider/data_service_provider.py

```diff
@@ -95,47 +96,53 @@ def encrypt(
 
     @staticmethod
     @enforce_types
-    def fileinfo(did: str, service_id: str, service_endpoint: str) -> Response:
-        payload = {"did": did, "serviceId": service_id}
+    def fileinfo(did: str, service: Any) -> Response:  # intermediary tweak
```

Comment:
Please remove comment.  I don't think this is intermediary anymore.

Reply:
Also, please change the service type to `Service`

Reply:
I discussed with @calina-c today, I cannot import Service class into DSP due to circular imports.